Client/Impl/Commands/UpdateRetriesCommand.cs

```diff
@@ -35,9 +36,9 @@ public async Task<IUpdateRetriesResponse> Send(TimeSpan? timeout = null, Cancell
             return new UpdateRetriesResponse();
         }
 
-        public async Task<IUpdateRetriesResponse> Send(CancellationToken cancellationToken)
+        public async Task<IUpdateRetriesResponse> SendWithRetry(CancellationToken cancellationToken)
```

Comment:
It seems you removed the send method, could you please bring it back, such that we have two sends and one sendWithRetry. Like here https://github.com/camunda-community-hub/zeebe-client-csharp/blob/master/Client/Impl/Commands/CompleteJobCommand.cs#L49-L64

Reply:
brought back the send so that there are two sends and a send with retry